substrate/client/transaction-pool/tests/integration.rs

```diff
@@ -58,28 +51,40 @@ async fn send_future_and_then_ready_from_many_accounts() {
 	// Shared params.
 	let send_threshold = 20_000;
 	let ws = "ws://127.0.0.1:9933";
```

Comment:
Can we get this URL from zn API? (if we update toml this can diverge).

Reply:
cc: @pepoviola

Reply:
Yes, done here: [9a665b6](https://github.com/paritytech/polkadot-sdk/pull/7257/commits/9a665b669efab2e2fb89c836be62ade310eb695f)